packages/gatsby-image/src/index.js

```diff
@@ -486,6 +487,7 @@ class Image extends React.Component {
           }}
           ref={this.handleRef}
           key={`fluid-${JSON.stringify(image.srcSet)}`}
+          {...tagProps}
```

Comment:
This should be the first prop, place it above all others, otherwise it introduces the possibility to override all prior props for this component.